dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -107,6 +109,7 @@ const SavedRunsComponent = () => {
                       selectAllRuns(isSelecting),
                     isSelected: areAllRunsSelected,
                   }}
+                  style={{ borderTop: "1px solid #d2d2d2" }}
```

Comment:
Just out of curiosity, why are we specifying the `style` in-line instead of putting it in the `.less` file?